cmd/controller/controller.go

```diff
@@ -349,8 +352,10 @@ func extractSiteRecords(status network.NetworkStatusInfo) []skupperv1alpha1.Site
 
 				if site, ok := routerAPs[link.Peer]; ok {
 					record.Links = append(record.Links, skupperv1alpha1.LinkRecord{
-						Name:         link.Name,
-						RemoteSiteId: site,
+						Name:           link.Name,
+						RemoteSiteId:   site,
+						RemoteSiteName: siteNames[site],
+						Operational:    strings.EqualFold(link.Status, "up"),
```

Comment:
Looks like we probably want to look at line 349 as well!

Reply:
Yes, thanks, I missed that! Fixed now.